code/modules/shuttle/shuttle.dm

```diff
@@ -410,6 +414,14 @@
 	initial_engines = count_engines()
 	current_engines = initial_engines
 
+	//Launch Announcement Alarm variables setup
+	alarm_sound_loop = new(src)
+	alarm_sound_loop.mid_length = 20
+	alarm_sound_loop.extra_range = 30
+	alarm_sound_loop.volume = 100
+	alarm_sound_loop.is_sound_projecting = TRUE
```

Comment:
Even with this set FALSE (I'm testing this with 4857 committed), I can still hear the alarm (quietly) from the beds in medbay. Not really able to discern any difference testing again with it back TRUE. Are you sure this works? Maybe confirm again yourself and make a video demonstrating what actually is changed?

Reply:
It does work. 


https://github.com/cmss13-devs/cmss13/assets/142365554/a077b8e9-5e83-4e40-9eaa-61ae09307110


https://github.com/cmss13-devs/cmss13/assets/142365554/9723b8ea-b956-456a-ac3c-88efa29c252d

Reply:
This is not what I experienced in testing. Standing on the vent one tile north of the bed in medbay it is equally audible regardless of whether the line 422 in shuttle.dm is set TRUE or FALSE.

HOWEVER, this appears to be because of the merge conflict in the file _looping_sound.dm that elects to automerge out the altered code: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/063fe80d-45f8-470a-a4e4-ad595362c5d3) (The code it needed user intervention on was the comments). The reason for this I believe is because commit 360c897cbf44c96799a19d74160c7acb17a7e119 committed the file unaltered so at least my git is treating the file as altered to be back to its original form.

This likely will break just like I am experiencing when that PR is merged into master. Though this weird merge behavior might change when the merge squashes the commits into a single commit. When you update w/ master be sure to closely examine the changes made by the merge. **In the future, I do not recommend making PRs that are dependent on other PRs.**

But yes the sound is *much* louder when this is working.